packages/xo-web/src/xo-app/backup/file-restore/restore-file-modal.js

```diff
@@ -75,15 +75,17 @@ const formatFilesOptions = (rawFiles, path) => {
 
 export default class RestoreFileModalBody extends Component {
   state = {
+    format: 'tgz',
     selectedFiles: [],
```

Comment:
shouldn't default format be zip ?

Reply:
I think it makes sense to use the best format by default.

We'll change it if we get negative comments about it.

Would that be good for you?